test/html/cases/markup.mli

```diff
@@ -88,6 +88,14 @@
           ignore foo
     ]}
 
+    Code blocks can have an identifier attached to them:
+
+    {ocaml[
```

Comment:
Misleading example? Using `ocaml` as the identifier makes it looks like an annotation specifying the language, rather than an identifier to enable references to this specific block from elsewhere.
```suggestion
    {print-two-plus-two[
```

Reply:
The current definition of annotations needs to be changed in my opinion as it has some limitations. Unfortunately I haven't been available to work on this in the last few months.

To clarify the misunderstanding here: the code block annotation is _not_ used to reference a specific block from elsewhere (it is not a unique identifier). It annotates a class of blocks to be extracted. It could be a language or a filename, or any other user-defined "class".

Not sure if this is helpful, but let me know if you have any other questions or suggestions.